src/components/elements/typography.ts

```diff
@@ -85,7 +100,13 @@ export const Text = styled.p`
 /////////////////// TEXT IS DEPRECATED. PLEASE USE HEADER. ///////////////////
 //////////////////////////////////////////////////////////////////////////////
 
-export const Header = styled(({ as = 'h2', children, ...props }) =>
+type HeaderProps = {
+    as?: string
+    children?: React.ReactNode
+    props?: TextProps
+} & TextProps
+
+export const Header = styled(({ as = 'h2', children, ...props }: HeaderProps) =>
```

Comment:
We are getting typescript errors in many files where we used `<Header>` component.
Try to fix those errors.

Reply:
Already resolved. Just forgot to add generic for styled component after props. Let's check again 🙏

Reply:
I still see the errors.
check these files: 
`src/pages/endpoint/index.tsx`
`src/pages/home/_hero.tsx`